src/openforms/formio/tests/test_dynamic_config.py

```diff
@@ -0,0 +1,292 @@
+from unittest.mock import Mock, patch
+
+from django.test import TestCase
+
+from rest_framework.test import APIRequestFactory
+
+from openforms.config.models import GlobalConfiguration
+from openforms.config.tests.factories import MapTileLayerFactory
+from openforms.formio.datastructures import FormioConfigurationWrapper
+from openforms.formio.dynamic_config import (
+    rewrite_formio_components,
+    rewrite_formio_components_for_request,
+)
+from openforms.submissions.tests.factories import SubmissionFactory
+
+rf = APIRequestFactory()
+
+
+class DynamicConfigTests(TestCase):
+    @patch("openforms.formio.components.vanilla.GlobalConfiguration.get_solo")
+    def test_map_without_default_map_config(self, m_solo: Mock):
+        m_solo.return_value = GlobalConfiguration(
+            form_map_default_zoom_level="8",
+            form_map_default_latitude="55.123",
+            form_map_default_longitude="56.456",
+        )
+
+        configuration = {
+            "components": [
+                {
+                    "type": "map",
+                    "key": "map",
+                    "defaultZoom": "3",
+                    "initialCenter": {
+                        "lat": "43.23",
+                        "lng": "41.23",
```

Comment:
are these stored as string or floats with the actual formio-builder? The types say `number`: https://github.com/open-formulieren/types/blob/5e2e2a219b4280c162c6eeffa6a51e333706bf49/src/formio/components/map.ts#L40

Reply:
Ah check, yeah these should be floats. Changing it 👍